Objects/TI4_Helpers/TI4_ExploreHelper.ttslua

```diff
@@ -149,6 +149,23 @@ function explorePlanet(tileGUID, planetName)
     broadcastToAll('Exploring Planet: ' .. planetName)
 end
 
+local _frontierTokenBagObject = nil
```

Comment:
Out of paranoia, I like caching via guid:
`local _frontierTokenBagGuid = false
...
local bag = getObjectByGuid(_frontierTokenBagGuid)
if bag then ...`

Reply:
Ah, that's a good way of doing it